Build/README.md

```diff
@@ -49,7 +49,7 @@ For the first setup, you will want to run this command:
 
 `cmake -G {YourGeneratorHere} .. && cmake ..`
 
-Replace {YourGeneratorHere} with one of the generator choices from `cmake --help`. As an example, Mac OS X users that want to have Xcode used would run `cmake -G Xcode .. && cmake ..` on their Terminal program.
+Replace {YourGeneratorHere} with one of the generator choices from `cmake --help`.
```

Comment:
I think it might nice to have some copy pastable commands, with the generators preset (visual studio for windows, unix makefiles for linux, xcode for mac).

But not necessary btw. Just a thought.

Reply:
I think we should encourage users to only use the default generator for their platform (VS for windows and unix makefiles for linux/macOS). Reason being that not all generators have all capabilities. e.g. The external project module we use for ffmpeg is not supported by ninja. IMO having more generators available has little benefit, but a lot of potential to cause build issues and confusion.